package.json

```diff
@@ -14,6 +14,7 @@
     "@storybook/addon-essentials": "^6.0.21",
     "@storybook/addon-links": "^6.0.21",
     "@storybook/react": "^6.0.21",
+    "@storybook/storybook-deployer": "^2.8.6",
```

Comment:
You'll need to add the dependency like so: `yarn add @storybook/storybook-deployer`. This will update yarn's lockfile (`yarn.lock`). Make sure to `git add` that file once its changed.

Reply:
I added the dependency and used `git add` to add `yarn.lock` to the commit.

Reply:
For clarification, did I need to do `yarn add` to make this repository compatible with developers using Yarn? I'm just making sure I'm understanding the lines I missed.

Reply:
Yup, that’s correct. It updates the lock file

Reply:
Okay, got it. Thanks for the clarification.